packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -719,8 +719,7 @@
     }
     &__text {
         position: absolute;
-        color: var(--status-success);
+        color: var(--status-success) !important;
```

Comment:
Is there any other workarounds than appending !important inside of text 🤨

Reply:
https://github.com/binary-com/deriv-app/pull/15064#discussion_r1594980660

Reply:
@aizad-deriv  Just commented here

Reply:
Hi @shahzaib-deriv I have an idea. Can we try this under &__container ?

.dc-tick-counter-bar__text {
            position: absolute;
            color: var(--status-success);
            width: 100%;
  }

Reply:
Updated @habib-deriv 🚀

Reply:
Text component has `styles` property. Would it be possible to pass it via property instead of passing it here? I believe it's a better way to make sure the color changes in the correct place.

Reply:
@akmal-deriv You're right. It should be like that, hoever text component color variable is different `var(--text-color)` than the one used here `--status-success`. in that case we might need to test wherever the Text component is being used after making a change to heavily used `Text` compnent

Reply:
After looking through defined css variables, the correct one for this usecase in `--text-profit-success` and I've updated the code to use the prop by removing the scss property